packages/shorebird_cli/pubspec.yaml

```diff
@@ -9,7 +9,15 @@ environment:
   sdk: ">=3.0.0 <4.0.0"
 
 dependencies:
-  archive: ^3.3.6
+  # We're using a specific commit because the latest published version (3.4.10)
+  # does not include a sync close API for ZipFileEncoder.
+  # https://github.com/brendan-duncan/archive/commit/94bf62410e9a073451bda0d851a07e8b8863bde4
+  # This was causing a flaky test on windows when using `shorebird preview`
+  # https://github.com/shorebirdtech/shorebird/issues/1902
+  archive:
+    git:
+      url: https://github.com/brendan-duncan/archive
```

Comment:
Nit since this might be exaggerated concerns: Usually when I stumble in cases like this, I tend to fork the repo under a org that I have control, just to be safe that this commit will not vanish.

I know the odds of that happening is not much though, but it don't hurt to be overzealous haha

Reply:
Yeah I thought about that but it seemed like overkill. Ideally we'll move away from this as soon as the next version is published.